src/components/table/table.stories.tsx

```diff
@@ -13,15 +15,30 @@ export default {
       },
     },
   },
-  argTypes: {
-    headers: {
-      control: 'array',
-      description:
-        'Use this prop to set the order of columns and assign headers. If none is provided, headers will be taken from the keys of the data objects from the `data` prop array and the first letter will be capitalized. \n<pre>```interface TableHeader {\n  dataKey: string\n  label?: string\n}```</pre>',
+}
+
+meta.argTypes = {
+  ...meta.argTypes,
+  headers: {
+    table: {
+      type: {
+        summary: 'Array of TableHeader.',
+        detail:
+          'Each TableHeader has the following fields:\n' +
+          '  dataKey: Field in table data for this header. \n' +
+          '  label: Optional label for this header.\n',
+      },
+      defaultValue: {
```

Comment:
I am wondering if we should avoid using `defaultValue` to explain types, as we may want to use it in the future to indicate the default value. Could we move it to the type detail section?

Reply:
This is for describing the default value. We are not explaining the type here.